docs/reference/reference-training/training/api-pytorch-trainer-reference.rst

```diff
@@ -0,0 +1,121 @@
+###############################
+ PyTorch Trainer API Reference
+###############################
+
```

Comment:
i moved this content into api-pytorch-reference.rst (so we don't need this extra file)

via a commit that i don't see on this PR
commit SHA f1a1e8513949d383de5622c9538c37355424ff63